html/browsers/origin/cross-origin-objects/cross-origin-objects-exceptions.html

```diff
@@ -1,333 +0,0 @@
-<link rel="help" href="https://html.spec.whatwg.org/multipage/#security-window">
-<link rel="help" href="https://html.spec.whatwg.org/multipage/#security-location">
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/common/get-host-info.sub.js"></script>
-<div id=log></div>
-<iframe id="B"></iframe>
-<iframe id="C"></iframe>
-<script>
-/*
- * Setup boilerplate. This gives us a same-origin window "B" and a cross-origin
- * window "C".
- */
-var host_info = get_host_info();
-
-setup({explicit_done: true});
-path = location.pathname.substring(0, location.pathname.lastIndexOf('/')) + '/frame.html';
-var B = document.getElementById('B').contentWindow;
-var C = document.getElementById('C').contentWindow;
-B.frameElement.uriToLoad = path;
-C.frameElement.uriToLoad = get_host_info().HTTP_REMOTE_ORIGIN + path;
-
-function reloadSubframes(cb) {
-  var iframes = document.getElementsByTagName('iframe');
-  iframes.forEach = Array.prototype.forEach;
-  var count = 0;
-  function frameLoaded() {
-    this.onload = null;
-    if (++count == iframes.length)
-      cb();
-  }
-  iframes.forEach(function(ifr) { ifr.onload = frameLoaded; ifr.setAttribute('src', ifr.uriToLoad); });
-}
-function isObject(x) { return Object(x) === x; }
-
-/*
- * Note: we eschew assert_equals in a lot of these tests, since the harness ends
- * up throwing when it tries to format a message involving a cross-origin object.
- */
-
-var testList = [];
-function addTest(fun, desc) { testList.push([fun, desc]); }
-
-
-/*
- * Basic sanity testing.
- */
-
-addTest(function() {
-  // Note: we do not check location.host as its default port semantics are hard to reflect statically
-  assert_equals(location.hostname, host_info.ORIGINAL_HOST, 'Need to run the top-level test from domain ' + host_info.ORIGINAL_HOST);
-  assert_equals(get_port(location), host_info.HTTP_PORT, 'Need to run the top-level test from port ' + host_info.HTTP_PORT);
-  assert_equals(B.parent, window, "window.parent works same-origin");
-  assert_equals(C.parent, window, "window.parent works cross-origin");
-  assert_equals(B.location.pathname, path, "location.href works same-origin");
-  assert_throws("SecurityError", function() { C.location.pathname; }, "location.pathname throws cross-origin");
-  assert_equals(B.frames, 'override', "Overrides visible in the same-origin case");
-  assert_equals(C.frames, C, "Overrides invisible in the cross-origin case");
-}, "Basic sanity-checking");
-
-/*
- * Whitelist behavior.
- *
- * Also tests for [[GetOwnProperty]] and [[HasOwnProperty]] behavior.
- */
-
-var whitelistedWindowIndices = ['0', '1'];
-var whitelistedWindowProps = ['location', 'postMessage', 'window', 'frames', 'self', 'top', 'parent',
-                              'opener', 'closed', 'close', 'blur', 'focus', 'length'];
-whitelistedWindowProps = whitelistedWindowProps.concat(whitelistedWindowIndices);
-whitelistedWindowProps.sort();
```

Comment:
...no, sorry. This is actually `whitelistedWindowPropNames`. Sorting the symbols throws.